src/main/java/org/opentripplanner/routing/alertpatch/AlertPatch.java

```diff
@@ -268,6 +231,10 @@ public int getDirectionId() {
         return directionId;
     }
 
+    public boolean hasDirectionId() {
+        return directionId != -1;
```

Comment:
Same comment as https://github.com/ibi-group/OpenTripPlanner/pull/55/files#r632083309 (i.e., would it be worth using a constant for this -1 value?).

Reply:
Refactored in d074833db78929d4776ad0ac37724965657cead7